integration-tests/chopsticks/src/network/polkadot.ts

```diff
@@ -0,0 +1,38 @@
+import { setupContext, SetupOption } from '@acala-network/chopsticks-testing'
+import type { Config } from './types.js'
+import { initialBalanceDOT, toNumber } from '../utils.js'
+
+export const options: SetupOption = {
+	endpoint: process.env.POLKADOT_WS || [
+		'wss://rpc.polkadot.io',
+		'wss://polkadot-rpc.dwellir.com',
+		'wss://rpc.ibp.network/polkadot',
+	],
+	db: './db/polkadot.db.sqlite',
+	port: toNumber(process.env.POLKADOT_PORT) || 9000,
+}
+
+export function setAddrNativeTokens(addr: string[], balance: bigint = initialBalanceDOT) {
```

Comment:
Can you name it the same as elsewhere?

Reply:
updated in: [bec955e](https://github.com/KILTprotocol/kilt-node/pull/614/commits/bec955eb1a2f911de2ef4479826913cacd5e5638)